indexer/feature_utils.cpp

```diff
@@ -427,4 +427,21 @@ vector<string> GetLocalizedRecyclingTypes(TypesHolder const & types)
   auto const & isRecyclingType = ftypes::IsRecyclingTypeChecker::Instance();
   return GetLocalizedTypes(isRecyclingType, types);
 }
+
+bool HasAtm(TypesHolder const & types)
+{
+    auto const & c = classif();
+    for (auto const t : types)
+    {
+      if (c.GetReadableObjectName(t) == "amenity-atm")
```

Comment:
That's not the fastest way to compare type.

Reply:
Changed to use `IsATMChecker`. I think it should be better